app/models/agents/gap_detector_agent.rb

```diff
@@ -8,6 +8,11 @@ class GapDetectorAgent < Agent
       The `value_path` value is a [JSONPath](http://goessner.net/articles/JsonPath/) to a value of interest. If either
       this value is empty, or no Events are received, during `window_duration_in_days`, an Event will be created with
       a payload of `message`.
+
+      The `stream_path` value is a [JSONPath](http://goessner.net/articles/JsonPath/) to a value in the event that will
```

Comment:
It would require code changes so that the old behavior is preserved when `detect_gaps_in_values` is not defined or `false`.
I find having `value_path` and `stream_path` as two different options confusing because the `path` itself can be the same of both behaviors, as I see it the difference is if the Agent only checks if the path is present in received events or checks for gaps in the received values at the path.